mne/channels/channels.py

```diff
@@ -440,18 +440,36 @@ def inst_has(attr):
 class InterpolationMixin(object):
     """Mixin class for Raw, Evoked, Epochs
     """
-    def interpolate_bads_eeg(self):
-        """Interpolate bad channels
 
-        Operates in place.
+    def interpolate_bads(self, reset_bads=True):
```

Comment:
That's cool, thanks!

2015-04-28 17:12 GMT+02:00 Alexandre Gramfort <notifications@github.com>:

> In mne/channels/channels.py
> <https://github.com/mne-tools/mne-python/pull/2011#discussion_r29252567>:
>
> >
> > -        Operates in place.
> > +    def interpolate_bads(self, reset_bads=True):
>
> fine with me so you can have in the example: ```evoked_after =
> evoked_before.copy().interpolate_bads()``` don't forget to explicit in the
> docstring that the instance is modified inplace.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2011/files#r29252567>.
>